src/Containers/FormContainer.tsx

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { View, StyleSheet, Text, Animated } from "react-native";
+import * as Icons from "../Components/Icons";
+import Colors from "../Themes/Colors";
+import Input from "../Components/Input";
+import { GradientButton, TransparentButton } from "../Components/Buttons";
+
+export type Props = {
+  emailLabel: string;
```

Comment:
I do not understand why do we need this component in ui-kit.

It is just a login form, it is not reusable for other forms.
And I do not understand zone of responsibility of this component.

Reply:
yes, it's login Form and it's reusable just like login form. So we can use one component in our apps, and it's terrible to create reusable form for all-purpose, I never do it

Reply:
Then it should be call LoginFormContainer

Reply:
and it's reusable for mobile and tablet devices, and for different themes

Reply:
Yes, inside `uikit` LoginForm it's a container which consists other components and some logic, but outside, in some app it's just LoginForm Component with props, that we add in LoginFormContainer render method and add some logic. But we don't call components like LoginFormComponent we call like LoginForm, or Button, or Input etc

Reply:
Anyway it is not a generic Form, it is LoginForm.

So it should be named properly and moved to some  folder, e.g. Forms

Reply:
that's why it's called `LoginForm`

Reply:
But I see file called `src/Containers/FormContainer.tsx`